src/views/v2/month/calendar-body/day/multiday-events/multiday-event/hidden.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/**
+ * View: Month View - Multiday Event Hidden
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/events/v2/month/calendar-body/day/multiday-events/multiday-event/hidden.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link {INSERT_ARTCILE_LINK_HERE}
+ *
+ * @version TBD
+ *
+ * @var WP_Post $event An event post object with event-specific properties added from the the `tribe_get_event`
+ *
+ * @see tribe_get_event() For the format of the event object and its properties.
+ *
+ */
+
```

Comment:
* :no_entry_sign: There must be exactly one blank line after the file comment
* :no_entry_sign: Additional blank lines found at end of doc comment

<sub>hash: 006657a408e8b78f535c499fbaf00edc</sub>